### PR TITLE
First desktop entries

### LINKED ISSUE #24: Desktop entries
For packages:
- eclipse-installer
- IntelliJ IDEA
- pycharm

https://standards.freedesktop.org/desktop-entry-spec/latest/

### PR DESCRIPTION
**Pull request by IanDuncanT**
Closes #24

### CODE REVIEW

File: desktop_entries/eclipse.desktop [hunk 1]
```diff
@@ -0,0 +1,9 @@
+[Desktop Entry]
+Type=Application
+Name=Eclipse
+Comment=Eclipse Integrated Development Environment
+Icon=~/eclipse-installer/eclipse-inst/icon.xpm
```
Comment (Wabri) [inline desktop_entries/eclipse.desktop]: The installation create this path? is so repetitive. if this is it i can edit the eclipse-install script.

File: desktop_entries/intelliJ.desktop [hunk 2]
```diff
@@ -0,0 +1,9 @@
+[Desktop Entry]
+Version=13.0
+Type=Application
+Terminal=false
+Icon[en_US]=~/.intellij-13/bin/idea.png
```
Comment (Wabri) [inline desktop_entries/intelliJ.desktop]: ec_US need to be there?

**Comment by IanDuncanT** [comment]
I am sorry about the ricebot.py That was for a discord bot for a friend, and it got removed when reinstalling raspbian on a pi.

File: desktop_entries/intelliJ.desktop (hunk 2 above)
Comment (IanDuncanT) [reply desktop_entries/intelliJ.desktop]: That is only for language if there is different logos. I don't know if there is one. It should be fine to remove.

File: desktop_entries/eclipse.desktop (hunk 1 above)
Comment (IanDuncanT) [reply desktop_entries/eclipse.desktop]: I don't know if it installed elsewhere, but that would be the default if under ~/

Comment (Wabri) [reply desktop_entries/eclipse.desktop]: The Yapi install eclipse-installer in home directory of user, so the ~ path is right

**Pull request merged**